modules/nw-external-dns-operator-configuration-parameters.adoc

```diff
@@ -24,22 +24,22 @@ spec:
       credentials:
         name: aws-access-key <2>
 ----
-<1> Defines available options such as AWS, GCP and Azure.
-<2> Defines a name of the `secret` which contains credentials for your cloud provider.
+<1> Defines available options such as AWS, GCP, and Azure.
```

Comment:
What are all of the available types? Isn't `Infoblox` also included?

Reply:
Added Infloblox